src/main/scala/scalismo/common/UnstructuredPoints.scala

```diff
@@ -65,6 +65,14 @@ sealed abstract class UnstructuredPoints[D: NDSpace: Create] private[scalismo] (
 
   }
 
+  override def pointsInChunks(nChunks: Int): Seq[Iterator[Point[D]]] = {
+    val chunkSize = pointSequence.size / nChunks
```

Comment:
Shouldn't this be floating point arithmetics and then ceiled? Otherwise, you will have to gather the last few elements in an additional chunk.

Reply:
you are totally right. This was a beginner's mistake. Should be fixed now